src/lib/factories.js

```diff
@@ -105,8 +114,8 @@ export function createShorthandFactory(Component, mapValueToProps, generateKey)
     throw new Error('createShorthandFactory() Component must be a string or function.')
   }
 
-  return (val, defaultProps) => {
-    return createShorthand(Component, mapValueToProps, val, defaultProps, generateKey)
+  return (val, defaultProps, overrideProps) => {
+    return createShorthand(Component, mapValueToProps, val, defaultProps, overrideProps, generateKey)
```

Comment:
This would also apply to the returned `create` factory.  Instead of:

```jsx
Menu.create(menu, null /* defaultProps */, () => /* override props */)
```

We'd have something like:

```jsx
Menu.create(menu, { overrideProps: () => { ... } })
```